tests/basic/cir-scalar-other-1-positive-out.yamlld

```diff
@@ -7,6 +7,8 @@
   - "@value": 123.45
   http://example.com/ystring:
   - "@value": string
+  http://example.com/ynull:
+  - "@value": null
```

Comment:
JSON-LD processors drop `null` values, so this would not be a valid expansion result.